dashboard/test/models/section_test.rb

```diff
@@ -66,13 +109,13 @@ class SectionTest < ActiveSupport::TestCase
     end
   end
 
-  test "cannot destroy section with students" do
+  test "can destroy section with students" do
```

Comment:
Note that the expected behavior tested here has changed as a result of switching `has_many :followers, dependent: :restrict_with_error` to has_many :followers, dependent: :destroy`.